From 3639736408e34291307c97aa66333a6d50b771ea Mon Sep 17 00:00:00 2001
From: Ofer Heifetz <oferh@marvell.com>
Date: Sun, 14 Dec 2014 16:08:59 +0200
Subject: [PATCH 0085/2241] bus: mvebu-mbus: not needed __init removal

	compiler section mismatch warning regarding __init.
	Removed the complained __init since it does not allocate
	any structures and does not need this annotation

	This commit fixes SYSTEMSW-1426

Signed-off-by: Ofer Heifetz <oferh@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/15314
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Nadav Haklai <nadavh@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/15403
(cherry picked from commit 69ce086522977e2697fb3d1236520f535b9f977b)
Reviewed-on: http://vgitil04.il.marvell.com:8080/16051
Reviewed-by: Neta Zur Hershkovits <neta@marvell.com>
Signed-off-by: Neta Zur <neta@marvell.com>

Change-Id: I7e57cec321c6d07c6189f51b38b157200f1f0ae4
Reviewed-on: http://vgitil04.il.marvell.com:8080/27318
Reviewed-by: Nadav Haklai <nadavh@marvell.com>
Tested-by: Ofer Heifetz <oferh@marvell.com>
---
 drivers/bus/mvebu-mbus.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/bus/mvebu-mbus.c b/drivers/bus/mvebu-mbus.c
index 80981a7..0faba37 100644
--- a/drivers/bus/mvebu-mbus.c
+++ b/drivers/bus/mvebu-mbus.c
@@ -1237,7 +1237,7 @@ static int __init mbus_dt_setup_win(struct mvebu_mbus_state *mbus,
 	return 0;
 }
 
-static int __init
+static int
 mbus_parse_ranges(struct device_node *node,
 		  int *addr_cells, int *c_addr_cells, int *c_size_cells,
 		  int *cell_count, const __be32 **ranges_start,
-- 
2.7.4

